### ISSUE TITLE
EXC_BREAKPOINT: _existingView > frame > horizontalSizeClass > sizeForChildContentContainer:withParentContainerSiz...

### ISSUE THREAD

**Issue by jkmassel** [wordpress-mobile/WordPress-iOS#15338]
**Sentry Url**: https://sentry.io/share/issue/9fdd1b49f1de421db6a24bc7d220383c/
**User Count**: 0
**Count**: 242
**First Release**: 16.0.0.6
**First Seen**: 2020-11-04T15:05:48Z
**Last Seen**: 2020-11-19T01:53:37Z
**24 Hours**: 18
**30 Days**: 236
**LogID:** EE8B041D-A763-4662-86A0-F02A392B468B

```
EXC_BREAKPOINT: _existingView > frame > horizontalSizeClass > sizeForChildContentContainer:withParentContainerSiz...
  File "LoginPrologueViewController.swift", line null, in value
  File "LoginPrologueViewController.swift", line 547, in LoginPrologueViewController.setButtonViewMargins
  File "<compiler-generated>", line null, in LoginPrologueViewController.buttonViewLeadingConstraint.getter
  File "LoginPrologueViewController.swift", line 547, in LoginPrologueViewController.setButtonViewMargins
  File "LoginPrologueViewController.swift", line 127, in LoginPrologueViewController.viewWillTransition
  File "<compiler-generated>", line 125, in LoginPrologueViewController.viewWillTransition
  File "WordPressAuthenticator.swift", line 177, in WordPressAuthenticator.showLogin
  File "WordPressAppDelegate.swift", line 608, in WordPressAppDelegate.showWelcomeScreen
  File "WordPressAppDelegate.swift", line 602, in WordPressAppDelegate.showWelcomeScreenIfNeeded
  File "<compiler-generated>", line null, in WordPressAppDelegate.showWelcomeScreenIfNeeded
  File "Classes/ViewRelated/Blog/Blog List/BlogListViewController.m", line 232, in -[BlogListViewController maybeShowNUX]
  File "Classes/ViewRelated/Blog/Blog List/BlogListViewController.m", line 157, in -[BlogListViewController viewWillAppear:]
  File "main.swift", line 7, in main
...
(78 additional frame(s) were not displayed)
```

**Comment by jkmassel** [comment]
https://sentry.io/share/issue/bf7246371a424e3ab5c9b7dfd41ec6e0/ seems like a very similar issue

**Comment by renanferrari** [comment]
Hey @ctarda, would you mind looking into this? It seems like you have worked around the area of the codebase that is coming up in the logs and would have the necessary context to better evaluate the issue.

I'm labeling this issue as medium priority ([high severity, low impact](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/docs/issue-triage.md#prioritization)) but feel free to change it as you see fit.

**Comment by ScoutHarris** [comment]
I attempted to fix this with https://github.com/wordpress-mobile/WordPress-iOS/pull/15232, but maybe that didn't work. I'll take this one.

**Comment by ScoutHarris** [comment]
I'm going to close this. The [fix](https://github.com/wordpress-mobile/WordPress-iOS/pull/15232) I implemented in in 16.2. Per this Sentry issue, this was last seen in release 16.0.0.6. If it reoccurs with/after 16.2 I'll take another look.
